### PR TITLE
Redis Health check support 

### PR DESCRIPTION
**Pull request by kumaramit01**
## 💸 TL;DR

Adds a Redis health check for baseplate.py. This can be used by redis envoy proxy.

## 📜 Details
[Jira] https://reddit.atlassian.net/browse/STRGS-240 and https://reddit.atlassian.net/browse/STRGS-12

## 🧪 Testing Steps / Validation
I was able to test the function that I added by running a local redis node and by stopping the service and testing to make sure I can get socket time out. 

## ✅ Checks

- [ ] CI tests (if present) are passing
- [ ] Adheres to code style for repo
- [ ] Contributor License Agreement (CLA) completed if not a Reddit employee

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -11,6 +11,10 @@ WORKDIR /src
 # don't care for our tests.
 ENV CASS_DRIVER_NO_EXTENSIONS theytaketoolongtobuild
 
+RUN apt-get update -y
```
Comment (kumaramit01) [inline Dockerfile:14]: FYI -I needed to add these for the building the kafka connector -there were header files missing and connector could not be built without the librdkafka-dev

**Review by nsheaps — CHANGES_REQUESTED** [review]
Rejecting until we have agreement on the PR that this PR uses functionality from

**Review by kylelemons — CHANGES_REQUESTED** [review]
Pending internal agreement on design.

(please don't link to internal documents from public PRs)

**Comment by kumaramit01** [comment]
We are changing the way this particular feature will be implemented in infrared -thanks for looking at this PR, closing this PR.

**Pull request closed without merging**